### ISSUE TITLE
Doesn't actually work

### ISSUE THREAD

**Issue by ylluminarious** [3DIndian/quora-upvotes#1]
I installed this extension and it seems that it doesn't actually work. The only thing that happens is that a `Sort by Votes` button appears as shown in your screenshot. But when I click that button, nothing happens. I don't even see any errors in the web console. I'm on Firefox 58.0.2, the most recent stable release.

**Comment by 3DIndian** [comment]
Yes, I haven't maintained it in a while. If you want to help maintain it, feel free to fork. 
I will see after I return to institute.
